k/keyring/keyring_ubi_9.3.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : keyring
+# Version       : 25.2.0
+# Source repo : https://github.com/jaraco/keyring
+# Tested on     : CentOS 7
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Sai Kiran Nukala <sai.kiran.nukala@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on the given
+#             platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such a case, please
+#             contact the "Maintainer" of this script.
+#
+# -----------------------------------------------------------------------------
+ 
+# Variables
+PACKAGE_NAME=keyring
+PACKAGE_VERSION=25.2.0
+PACKAGE_URL=https://github.com/jaraco/keyring
+TARBALL_URL=https://files.pythonhosted.org/packages/b8/09/fdd3a390518e3aebeec0d7aceae7f9152da1fd2484f12f1b3a12a74aa079/keyring-25.2.0.tar.gz
+ 
+# Install dependencies
+yum install -y --allowerasing yum-utils git gcc gcc-c++ make curl python3 python3-pip python3-devel openssl-devel pkg-config
+ 
+# Install Rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source "$HOME/.cargo/env"
+ 
+# Create and activate Python virtual environment
+python3 -m venv keyring-env
```

Comment:
@saiNammi why we required venv?

Reply:
Initially when I was working on ubuntu machine I have created separate env for every package which i was working on.
I have removed that thing and updated script.